sonarqube/iap_proxy.py

```diff
@@ -5,8 +5,8 @@
 import json
 from urllib.parse import quote as urlquote
 
-from google.oauth2.service_account import IDTokenCredentials
-from google.auth.transport.requests import Request
+from msrestazure.azure_active_directory import AADTokenCredentials
+import adal
```

Comment:
Don't we need to add these libraries to the docker container?

Reply:
Yes, added the 'msrestazure' library now. That should also install 'adal'.